docs/src/main/sphinx/admin/properties-general.rst

```diff
@@ -36,3 +36,24 @@ across nodes in the cluster. It can be disabled, when it is known that the
 output data set is not skewed, in order to avoid the overhead of hashing and
 redistributing all the data across the network. This can be specified
 on a per-query basis using the ``redistribute_writes`` session property.
+
+``protocol.v1.alternate-header-name``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Type:** ``string``
+
+The 353-e release of Trino changes the HTTP client protocol headers to start
+with ``X-Trino-``. Legacy clients can be temporarily supported by setting this
+configuration property to ``Presto``. The preferred approach to migrating from
+versions earlier than 353 is to update all clients together with the release, or
+immediately afterwards, and then remove usage of this property.
+
+Ensure to use this only as a temporary measure to assist in your migration
+efforts, for example if you continue to operate clusters with an older version
+of Trino.
+
+In addition, use a 350-e version of the relevant client tools for best
```

Comment:
350

Reply:
I kind of think this whole line 55 and 56 can go. We will have this in the migration guide later.